### DISCUSSION TITLE
ScriptReference/MonoBehaviour.OnMouseOver

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#73] [Notes]
# ScriptReference/MonoBehaviour.OnMouseOver

https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnMouseOver.html

**Comment by vertxxyz** [comment]
If you are using the new Input System, this is not called. (See [known limitations](https://docs.unity3d.com/Packages/com.unity.inputsystem@latest/index.html?subfolder=/manual/KnownLimitations.html).)
